umbral/openssl.py

```diff
@@ -63,13 +66,12 @@ def _bn_is_on_curve(check_bn, curve_nid: int):
     return check_sign == 1 and range_check == -1
 
 
-def _int_to_bn(py_int: int, curve_nid: int=None):
+def _int_to_bn(py_int: int, curve_nid: int=None, is_consttime=True):
```

Comment:
Nice work, how about the name `constant_time`? :1st_place_medal:

Reply:
Do you feel good about simply shortening it to `constant_time=True`?

`use_constant_time` kinda feels like it affects _everything_ as a BIGNUM, when it only affect a few operations.

Reply:
Well, conversely, `constant_time=True` feels like it only effects this instantiation, rather than the resulting object.